src/commands/publish.ts

```diff
@@ -0,0 +1,19 @@
+import { generateContractsForm } from '../utils/generate-contracts-form';
+import { CliArguments } from '../utils/define-args';
+import { readPacts } from '../utils/read-pacts';
+import { buildUrl } from '../utils/build-url';
+import { postPacts } from '../utils/postPacts';
+
+export async function publish(argv: CliArguments) {
+    const { pactsDir, serviceName, url, serviceVersion } = argv;
+
+    const pacts = readPacts(pactsDir);
+    const contractsForm = generateContractsForm(pacts);
+    const judgeDUrl = buildUrl(url, serviceName, serviceVersion);
+
+    try {
+        await postPacts(judgeDUrl, contractsForm);
+    } catch (e) {
+        console.log(e.message);
```

Comment:
This try/catch would cause process to exit with status 0 when there is error. Since errors are not handled in any way, I would just remove try/catch here, and handle rejected promise in main module